README.md

```diff
@@ -1,8 +1,73 @@
 # orb_slam_2_ros
-This package integrates orb_slam_2 into ROS in what I believe to be a more user friendly way than what is offered by the underlying library found here:
 
-https://github.com/raulmur/ORB_SLAM2
+## Overview
 
-Depends on my catkinized version of the orb_slam_2 library:
+This package integrates orb_slam_2 into ROS in a more user friendly way than what is offered by the original library [ORB_SLAM2](https://github.com/raulmur/ORB_SLAM2)
 
-https://github.com/ethz-asl/orb_slam_2_catkin
+**Keywords:** ROS, ORB_SLAM2, SLAM
+
+### License
+
+The source code is released under a [TODO @alexmillane insert license](put_link_here).
+
+**Author(s): Alex Millane
+Maintainer: Alex Millane
```

Comment:
Feel free to add yourself to these lists. By writing you're as much an author/maintainer as I am.

Reply:
Only if you actually want to of course.

Reply:
I don't think it's fair now. I have just written this readme file ;)

Reply:
also here you need a license :D

Reply:
Added. I have just added the same license as orb_slam.